spec/datadog/tracing/contrib/suite/integration_spec.rb

```diff
@@ -39,7 +39,12 @@
       Datadog.configure { |c| c.remote.poll_interval_seconds = 0.001 }
     end
 
-    after { WebMock.disable! }
+    after do
+      # Ensure RC background worker is stopped before we disable webmock
+      # to avoid failed HTTP requests, trying to make a real remote call.
+      Datadog.shutdown!
```

Comment:
This currently generates an error in a background thread, after the test assertions finish, because the RC worker thread tries to make an unmocked RC call after we call `WebMock.disable!`. This PR ensures the RC worker is closed before `WebMock.disable!` is called.